code/style.css

```diff
@@ -1,4 +1,300 @@
+  transform: scaleX(1);
+  transform-origin: left;
+}
+
+/*** LOGO AND HAMBURGER/NAVBAR ***/
+header {
+  background-color: rgba(255,255,255,0.5);
+  box-sizing: border-box; 
+  position: fixed;
+  top: 0;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  padding: 12px 30px;
+  box-shadow: 5px 5px 5px rgba(129, 129, 129, 0.8);
+}
+
+.header-logo {
+  margin-right: auto;
+}
+
+.header-logo img {
+  width: 100px;
+  margin-right: 10px;
+}
+
+nav {
+  display: none;
+}
+
+/*** BIG NEWS SECTION ***/
+.big-news-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 90%;
+  margin: 150px auto;
+  background: white;
+  border: 2px solid white;
+  box-shadow: 5px 5px 5px rgba(59, 59, 59, 0.8);
+  padding: 10px;
+}
+
+.big-news-container img {
+  width: 100%;
+}
+
+.big-news-container h3 {
+  font-size: 30px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  font-family: 'Source Code Pro', monospace;
+}
+/* LONG PARAGRAPH ONLY FOR TABLETS AND BIGGER */
+.big-news-long-paragraph {
+  display: none;
+}
+
+.big-news-textbox {
+  text-align: left;
+}
+
+/*** SMALL NEWS SECTION ***/
+.small-news-section-container {
+  margin: 10px auto 30px auto;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  width: 80%;
+}
+
+.small-news-container {
+  width: 90%;
+  text-align: left;
+  margin-bottom: 50px;
+  background: white;
+  border: 2px solid white;
+  box-shadow: 5px 5px 5px rgba(59, 59, 59, 0.8);
+  padding: 30px 30px 50px 30px;
+}
+
+.small-news-container img {
+  width: 100%;
+}
+
+.small-news-container h4 {
+  font-size: 30px;
+  font-weight: bolder;
+  margin-top: 15px;
+  margin-bottom: 0px;
+}
+
+.small-news-container p {
+  margin-top: -30px;
+}
+
+/*** FOOTER ***/
+footer {
+  display: flex;
+  justify-content: space-evenly;
+  background-color: rgba(255,255,255,0.5);
+}
+
+footer ul {
+  list-style-type: none;
+}
+
+footer a {
+  text-decoration: none;
+  color: black;
+}
+
+footer a:hover {
+  color: white;
+  transition: 0.3s;
+}
+
+footer ul img {
+  width: 15px;
+  height: 15px;
+  margin-right: 5px;
+}
+
+/*** MEDIA QUERIES TABLET ***/
+@media (min-width: 768px) {
+  /* NAV MENU REPLACES HAMBURGER */
+  nav {
+    display: inline-block;
+  }
+ 
+  nav a {
+    color: black;
+    text-decoration: none;
+    margin-left: 30px;
+  }
+
+  nav a:hover {
+    color: white;
+    font-weight: bold;
+    transition: 0.3s;
+  }
+
+  /*** BIG NEWS SECTION ***/
+  .big-news-container {
+    display: block;
+    overflow: auto;
+  }
+
+  .big-news-container img {
+    width: 35%;
+    float: left;
+    padding: 20px;
+  }
+
+  .big-news-textbox {
+    margin-left: 20px;
+    padding: 10px;
+    text-align: left;
+    color: black;
+  }
+
+  .big-news-long-paragraph {
+    display: block;
+  }
+
+  /* LONG PARAGRAPH REPLACES SHORT */
+  .big-news-short-paragraph {
+    display: none;
+  }
+
+  .big-news-container h3 {
+    margin-top: 3px;
+  }
```

Comment:
Not sure about this, but it looks like the word-spacing is huge on this one (causing the heading to wrap on two lines). Possibly try this: `word-spacing: -5px;` to shrink it

Reply:
Or just decrease the font-size a bit could of course also work 😅